wsmaster/che-core-api-workspace-activity/src/test/java/org/eclipse/che/api/workspace/activity/spi/tck/WorkspaceActivityDaoTest.java

```diff
@@ -186,6 +186,25 @@ public void shouldFindActivityByLastStatusChangeTime(WorkspaceStatus status) thr
     assertEquals(found.getItems(), singletonList(activities[0].getWorkspaceId()));
   }
 
+  @Test(dataProvider = "allWorkspaceStatuses")
+  public void shouldCountWorkspacesInStatus(WorkspaceStatus status) throws Exception {
```

Comment:
I'm OK with this test but just want to note that mostly we write our tests in `GivenWhenThen` manner while this method invokes `countWorkspacesInStatus` three times and tests different cases in one method.